packages/react-vapor/src/components/toast/Toast.tsx

```diff
@@ -12,6 +12,7 @@ export interface IToastProps {
     dismiss?: number;
     dismissible?: boolean;
     animate?: boolean;
+    isDownload?: boolean;
```

Comment:
I feel like download should juste be another possible value of the `type` prop.

Reply:
that makes more sense 🤦